lessons/en/basics/enum.md

```diff
@@ -1,13 +1,13 @@
 %{
-  version: "1.8.0",
+  version: "1.8.1",
```

Comment:
I think this should be 1.9 since it is a change we’d like to see made in the translations